python/ray/data/preprocessors/concatenator.py

```diff
@@ -6,43 +6,101 @@
 
 
 class Concatenator(Preprocessor):
-    """Creates a tensor column via concatenation.
+    """Combine numeric columns into a column of type
+    :class:`~ray.air.util.tensor_extensions.pandas.TensorDtype`.
 
-    A tensor column is a column consisting of ndarrays as elements.
-    The tensor column will be generated from the provided list
-    of columns and will take on the provided "output" label.
-    Columns that are included in the concatenation
-    will be dropped, while columns that are not included in concatenation
-    will be preserved.
+    This preprocessor concatenates numeric columns and stores the result in a new
+    column. The new column contains
+    :class:`~ray.air.util.tensor_extensions.pandas.TensorArrayElement` objects of
```

Comment:
seems like `TensorArrayElement` is not a documented class?

Reply:
It's not. If we add `TensorArrayElement` to the data reference in a future PR, this link will work.